### PR TITLE
Add SecurityZone stub

### PR DESCRIPTION
**Pull request by danmosemsft**
Relates to https://github.com/dotnet/corefx/issues/18078 (still needs CoreFX exposure)

Stub just like the other CAS related members.
@ViktorHofer

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@danmosemsft Thanks for adding! Will you also update the corefx refs after coreclr sync or should I?

**Comment by danmosemsft** [comment]
If you're offering -- sure, grab it. 
Signed off?

**Comment by ViktorHofer** [comment]
Sure.

File: src/mscorlib/shared/System/Security/SecurityException.cs [hunk 1]
```diff
@@ -62,5 +62,16 @@ protected SecurityException(SerializationInfo info, StreamingContext context)
         public object PermitOnlySetInstance { get; set; }
         public string RefusedSet { get; set; }
         public string Url { get; set; }
+        public SecurityZone Zone { get; set; }
```
Comment (ViktorHofer) [inline src/mscorlib/shared/System/Security/SecurityException.cs]: Not sure how much we care about alphabetical order

**Comment by ViktorHofer** [comment]
Just noticed that moving the enum into a separate file would have been better. (like other enums in that dir)

**Comment by weshaggard** [comment]
@danmosemsft where do you plan to put SecurityZone in the contracts? I don't think this type is worth exposing at all as it is essentially dead and will never work.

**Pull request merged**